src/spec/dsl.cr

```diff
@@ -64,6 +65,28 @@ module Spec
     exit
   end
 
+  # :nodoc:
+  class_getter randomizer_seed : UInt64?
+  class_getter randomizer : Random::PCG32?
+
+  # :nodoc:
+  def self.order=(mode)
+    seed =
+      case mode
+      when "default"
```

Comment:
Hmm I don't like the strings being used here. At least it could be symbols. Or change it to `seed=`, `random_seed`.

Reply:
The methods in this DSL module are basically the implementation of the CLI, so I'd like to keep it as close to the CLI as possible.  This includes incoming values as strings, and the method being named `order=` to match the `--order` CLI param.